storm-core/src/storm.thrift

```diff
@@ -244,8 +244,9 @@ struct CommonAggregateStats {
 2: optional i32 num_tasks;
 3: optional i64 emitted;
 4: optional i64 transferred;
-5: optional i64 acked;
-6: optional i64 failed;
+5: optional double throughput;
+6: optional i64 acked;
+7: optional i64 failed;
```

Comment:
This is another problem with maintaining binary compatibility with the previous code.  You cannot renumber entries.  The tags at the beginning are what identify the field in the binary data.  By renumbering them the new code and old code will mix up throughput, acked, and failed.